llvm/lib/Target/AArch64/MCTargetDesc/AArch64MCTargetDesc.cpp

```diff
@@ -352,6 +352,8 @@ static MCAsmInfo *createAArch64MCAsmInfo(const MCRegisterInfo &MRI,
     MAI = new AArch64MCAsmInfoMicrosoftCOFF();
   else if (TheTriple.isOSBinFormatCOFF())
     MAI = new AArch64MCAsmInfoGNUCOFF();
+  else if (TheTriple.isUEFI())
+    MAI = new AArch64MCAsmInfoGNUCOFF();
```

Comment:
We shouldn't be using GNU COFF for UEFI, we should be using Microsoft COFF.

Reply:
Oh, why does x86 use it?
https://github.com/llvm/llvm-project/blob/ee4c8b556c5cf42c55ce9540bbb0e29c11894a71/llvm/lib/Target/X86/MCTargetDesc/X86MCTargetDesc.cpp#L455-L456

Is it because of the win64 calling convention?